pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -1147,7 +1147,7 @@ func TestPolicyWithAlignBySocketAndDistributeCPUsAcrossNUMAEnabled(t *testing.T)
 	testCases := []staticPolicyTest{
 		{
 			description: "AlignBySocket and DistributeCPUsAcrossNUMA both enabled",
-			topo:        topoDualSocketDualNUMAPerSocketNoHT,
+			topo:        topoDualSocketMultiNumaPerSocketHTType2,
```

Comment:
perhaps rename `topoDualSocketMultiNumaPerSocketSublusterHT`? or in general something more descriptive than `Type2`